### PR TITLE
[nri-kube-events] Add custom attributes support through the agent

### PR DESCRIPTION
**Pull request by kang-makes**
#### Is this a new chart

No

#### What this PR does / why we need it:

Custom Attributes are needed for the Kubernetes rearchitecture so a patch to send Custom Attributes in every metric has been added to the agent: https://github.com/newrelic/infrastructure-agent/pull/825

This PR adds custom attributes to kube-events which currently is not supported.

#### Which issue this PR fixes
https://github.com/newrelic/helm-charts/issues/110

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by kang-makes** [comment]
Force push to rebase to the last commit of `master`.

Blocked by newrelic/infrastructure-agent#911

File: charts/nri-kube-events/templates/_helpers.tpl [hunk 1]
```diff
@@ -141,3 +141,31 @@ licenseKey and cluster are set.
 {{- $customSecretLicenseKey := include "nri-kube-events.customSecretLicenseKey" . -}}
 {{- and (or $licenseKey (and $customSecretName $customSecretLicenseKey)) $cluster}}
 {{- end -}}
+
+{{/*
+Return the cluster name
+*/}}
+{{- define "newrelic.cluster" -}}
+{{- if .Values.cluster -}}
+  {{- .Values.cluster -}}
```
Comment (paologallinaharbur) [inline charts/nri-kube-events/templates/_helpers.tpl:150]: Cluster name is populated already in a different way (I do not fully remember how).

I would avoid adding it also as a custom attribute. Therefore I would have merely customAttributesWithoutClusterName.

Comment (kang-makes) [reply charts/nri-kube-events/templates/_helpers.tpl:150]: You are right. Here: https://github.com/newrelic/helm-charts/pull/598/files/0ce03b02a64c24836fef4d55cbf3bea37bcaa068#diff-131da0b26df332b77efd10d0695ed21bff8234051410998ae18562a7d0a43b79R74-R87

And here: https://github.com/newrelic/helm-charts/blob/0ce03b02a64c24836fef4d55cbf3bea37bcaa068/charts/nri-kube-events/templates/configmap.yaml#L13

I am removing those changes.

**Review by paologallinaharbur — APPROVED** [review]

File: charts/nri-kube-events/templates/deployment.yaml [hunk 2]
```diff
@@ -85,6 +85,10 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: spec.nodeName
+            {{- if .Values.customAttributes }}
+            - name: NRIA_CUSTOM_ATTRIBUTES
+              value: {{ include "newrelic.customAttributes" . | quote }}
+            {{- end }}
```
Comment (roobre) [inline charts/nri-kube-events/templates/deployment.yaml:91]: I think the scope of `"newrelic.customAttributes"` could be reduced by passing just the customAttributes directly:

```suggestion
            {{- with .Values.customAttributes }}
            - name: NRIA_CUSTOM_ATTRIBUTES
              value: {{ include "newrelic.customAttributes" . | quote }}
            {{- end }}
```

WDYT?

Comment (kang-makes) [reply charts/nri-kube-events/templates/deployment.yaml:91]: This was approved on Wednesday and we said that this was going to be released today.

I am sorry to say that I am not going last-minute changes :(

**Pull request merged**